### PR TITLE
Satellite Projects db entity and api updated

### PR DESCRIPTION
**Pull request by crawter**
1. Added CompanyName field to Project entity
2. Project Name is not updatable now
3. Insert in repo and CreateProject in service updated to use CompanyName for project creation
4. GetUsersProjects method in service updated and now returns only projects in which current user participate
5. Updated frontend, to support CompanyName

### CODE REVIEW

File: pkg/satellite/satellitedb/dbx/satellitedb.dbx.go [hunk 1]
```diff
@@ -291,6 +291,7 @@ CREATE TABLE projects (
 	id BLOB NOT NULL,
 	owner_id BLOB REFERENCES users( id ) ON DELETE SET NULL,
 	name TEXT NOT NULL,
+	company_name TEXT NOT NULL,
```
Comment (crawter) [inline pkg/satellite/satellitedb/dbx/satellitedb.dbx.go]: i should double check it, it seems that company name could be nullable

File: pkg/satellite/satellitedb/projects.go [hunk 2]
```diff
@@ -39,6 +40,60 @@ func (projects *projects) GetByOwnerID(ctx context.Context, ownerID uuid.UUID) (
...
+						on pm.project_id = p.id
+							where pm.member_id = $1`, userID[:])
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		err = rows.Close()
+		if err != nil {
+			// TODO: Should another logger to be used here?
+			log.Fatal(err.Error())
```
Comment (egonelbre) [inline pkg/satellite/satellitedb/projects.go]: return the error
```
errors = append(errors, rows.Close())
return products, utils.CombineErrors(errors...)
```

File: pkg/satellite/satelliteweb/satelliteql/project.go [hunk 3]
```diff
@@ -43,6 +46,22 @@ func graphqlProject() *graphql.Object {
...
+					if project.OwnerID == nil {
+						return "", nil
+					}
+
+					user, err := service.GetUser(p.Context, *project.OwnerID)
+					if err != nil {
+						return "", nil
+					}
+
+					return fmt.Sprintf("%s %s", user.FirstName, user.LastName), nil
```
Comment (egonelbre) [inline pkg/satellite/satelliteweb/satelliteql/project.go]: return user.FirstName + " " + user.LastName, nil

File: pkg/satellite/satellitedb/projects.go [hunk 4]
```diff
@@ -39,6 +40,60 @@ func (projects *projects) GetByOwnerID(ctx context.Context, ownerID uuid.UUID) (
 	return projectsFromDbxSlice(projectsDbx)
 }
 
+// GetByUserID is a method for querying all projects from the database by userID.
+func (projects *projects) GetByUserID(ctx context.Context, userID uuid.UUID) ([]satellite.Project, error) {
+	var products []satellite.Project
+	var errors []error
+
+	rows, err := projects.db.Query(`
```
[the remark this answers is not in the document]
Comment (crawter) [reply pkg/satellite/satellitedb/projects.go]: not sure that its possible, i asked about this query in dbx slack channel and answer wasn't clear enough

File: web/satellite/src/api/projects.ts [hunk 5]
```diff
@@ -7,6 +7,8 @@ import gql from "graphql-tag";
 // Performs graqhQL request.
 // Throws an exception if error occurs
 export async function createProject(project: Project): Promise<any> {
+    console.log("in api", project);
```
Comment (egonelbre) [inline web/satellite/src/api/projects.ts]: This probably shouldn't be here.

**Pull request merged**